### ISSUE TITLE
Pico gets hot and starts smelling smokey when connected to battery 

### ISSUE THREAD

**Issue by BadRothenfelde** [veebch/photon#25]
is that normal?

**Comment by veebch** [comment]
No. Double check the polarity on the battery connector and the battery

**Comment by BadRothenfelde** [comment]
But I physically can't connect it the other way around since the clip from the battery doesn't fit  then.

**Comment by veebch** [comment]
I once had the same thing happen to me. Notice the warning (4th pic) [here](https://www.amazon.de/-/en/Battery-Lithium-Batteries-polarity-purchase/dp/B08FD39Y5R)

**Comment by ryan1894** [comment]
had same issue. you can lift a plastic tab on the battery white port and pull out the pin to swap it.

**Comment by BadRothenfelde** [comment]
Okay, cheers
